javascript/ql/lib/semmle/javascript/frameworks/SQL.qll

```diff
@@ -540,3 +598,145 @@ private module SpannerCsv {
     }
   }
 }
+
+/**
+ * Provides classes modelling the Google Cloud Spanner library.
```

Comment:
```suggestion
 * Provides classes modeling the Google Cloud Spanner library.
```
(as suggested by code scanning)

Reply:
*sigh* I'll need a spell checker for VS Code. Thx!